### ISSUE TITLE
Bagnon 9.0.0 Conflict with DurabilityAndItemLevel addone

### ISSUE THREAD

**Issue by gaveer** [enderneko/DurabilityAndItemLevel#1]
**Running Software (issues missing this information will be deleted):**
 - Addon version: Bagnon-9.0.0-9.0.1
 - Addon version: DurabilityAndItemLevel 3.3.0
 - Server patch: Retail 8.3.7.35662

**To Reproduce**
Steps to reproduce the behaviour:
1. Enter Wolrd
2. See error

**Error Logs**
``` 
1x ...faceDurabilityAndItemLevel\ItemLevel_Bag.lua:86: Usage: hooksecurefunc([table,] "function", hookfunc)
[string "=[C]"]: in function `hooksecurefunc'
[string "@DurabilityAndItemLevel\ItemLevel_Bag.lua"]:86: in main chunk
```

**Comment by gaveer** [comment]
@enderneko

**Comment by enderneko** [comment]
DurabilityAndItemLevel is currently incompatible with Bagnon 9.0, and do not work with 9.0.
I'll fix it when shadowlands pre-patch releases. I can't get access to shadowlands beta, cuz I play cnwow, and I don't have a international battlenet account.
I'm sorry, you may have to wait. Or you can try to use Bagnon 8.3 version in retail.

**Comment by enderneko** [comment]
Fixed in 3.3.1.
Thanks for your feedback.

**Comment by gaveer** [comment]
No Thank you
